### DISCUSSION TITLE
How to create two relations to same table in prisma schema?

### DISCUSSION THREAD

**Discussion by leciric** [prisma/prisma#9575] [Q&A] [answered]
Hello guys, I have two tables one named 'Offer' and other named 'User'. 
Offer have two fields to store two users id, one for a seller and one for a buyer.

But when I run this migration, prisma automatically creates UNIQUE INDEX to both of this fields, and I need to have more than one offers with same seller and buyer ids, but it's impossible this way.

Here are the schema:

```
model offer {
  id            String         @id @default(uuid()) @db.VarChar(255)
  value         Decimal        @db.Decimal(19, 2)

  created_at    DateTime       @default(now()) @db.Timestamp(0)
  updated_at    DateTime       @updatedAt @db.Timestamp(0)

  seller_id     String         @db.VarChar(255)
  buyer_id      String         @db.VarChar(255)

  buyer         user           @relation("buyer", fields: [buyer_id], references: [id])
  seller        user           @relation("seller", fields: [seller_id], references: [id])
}

model user {
  id            String             @id @default(uuid()) @db.VarChar(255)
  name          String             @db.VarChar(255)
  password      String?            @db.VarChar(255)
  email         String             @unique @db.VarChar(255)

  created_at    DateTime           @default(now()) @db.Timestamp(0)
  updated_at    DateTime           @updatedAt @db.Timestamp(0)

  buyer         offer?             @relation("buyer")
  seller        offer?             @relation("seller")
}
```

And here is what the migration generates every time:

```
    UNIQUE INDEX `offer_buyer_id_unique`(`buyer_id`),
    UNIQUE INDEX `offer_seller_id_unique`(`seller_id`),

```

I'd like to remove this UNIQUE from indexes, but when I remove prisma applies it again.

Found something similar on in this [issue](https://github.com/prisma/prisma/issues/5354#issuecomment-771065615)

**Comment by leciric** [comment, marked as answer]
I was creating a relation one to one, but I was thinking that was one to many, that's why this was happening.
Just found out.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
